recipes/cxxopts/all/test_package/main.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include <iostream>
+#include <iostream>
```

Comment:
FYI, this is because I removed the BOM (https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8)
```
00000000  ef bb bf 23 69 6e 63 6c  75 64 65 20 3c 69 6f 73  |...#include <ios|
00000010  74 72 65 61 6d 3e 0a 23  69 6e 63 6c 75 64 65 20  |tream>.#include |
00000020  3c 63 78 78 6f 70 74 73  2e 68 70 70 3e 0a 0a 69  |<cxxopts.hpp>..i|
00000030  6e 74 20 6d 61 69 6e 28  69 6e 74 20 61 72 67 63  |nt main(int argc|
00000040  2c 20 63 68 61 72 2a 20  61 72 67 76 5b 5d 29 0a  |, char* argv[]).|
00000050  7b 0a 09 63 78 78 6f 70  74 73 3a 3a 4f 70 74 69  |{..cxxopts::Opti|
00000060  6f 6e 73 20 6f 70 74 69  6f 6e 73 20 7b 20 22 74  |ons options { "t|
00000070  65 73 74 5f 70 61 63 6b  61 67 65 22 2c 20 22 54  |est_package", "T|
00000080  68 65 20 65 78 61 6d 70  6c 65 20 70 72 6f 67 72  |he example progr|
00000090  61 6d 20 74 6f 20 74 65  73 74 20 63 78 78 6f 70  |am to test cxxop|
```